bot/exts/fun/hangman.py

```diff
@@ -0,0 +1,140 @@
+from asyncio import TimeoutError
+from pathlib import Path
+from random import choice
+from typing import Literal
+
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours
+
+
+TOP_WORDS_FILE_PATH = Path("bot/resources/fun/top_1000_used_words.txt").resolve()
+# defining a list of images that will be used for the game to represent the hangman person
+IMAGES = [
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472355056295946/hangman0.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472756744814613/hangman1.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472808699629578/hangman2.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472862441267230/hangman3.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472950991396864/hangman4.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472999431430204/hangman5.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883473051277226015/hangman6.png",
+]
```

Comment:
Are you confident that these images won't be deleted from Discord?

Just mentioning as if they are, the cog will need to be updated, and the images may be lost.

Reply:
Yep, I uploaded these images and they are in my sir-lancebot testing server which I will always have, and if I ever do delete it I will migrate it to imgur

Reply:
yup no worries, just wanted to make you aware 👍